MismatchGeneration/utils.py

```diff
@@ -25,6 +32,10 @@
 
 logging.disable(logging.WARNING)
 os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+<<<<<<< HEAD
+=======
+import tensorflow as tf
+>>>>>>> c4a81f61b7b4fdf8f24ba9223274bb63279c9694
```

Comment:
Hey @LofiTea: quick note on this is that you have the merge conflicts now in the code :) Let's take a look at this later today!